examples/site/howto_offload_builder_creation/README.md

```diff
@@ -0,0 +1,148 @@
+# How-to Guide: Use Cloud Build to create a buildpacks builder
+
+[repository-gh]: https://github.com/GoogleCloudPlatform/functions-framework-cpp
+[cloud-build-quickstarts]: https://cloud.google.com/build/docs/quickstarts
+[gcp-quickstarts]: https://cloud.google.com/gcp/getting-started
+[buildpacks]: https://buildpacks.io
+[docker]: https://docker.com/
+[docker-install]: https://store.docker.com/search?type=edition&offering=community
+[sudoless docker]: https://docs.docker.com/engine/install/linux-postinstall/
+[pack-install]: https://buildpacks.io/docs/install-pack/
+[hello-world-http]: /examples/site/hello_world_http/hello_world_http.cc
+[cloud-build]: https://cloud.google.com/cloud-build
+
+This guide shows you how to use [Cloud Build][cloud-build] to offload the
+creation of [buildpacks] builders.
+
+## Pre-requisites
+
+This guide assumes you are familiar with Google Cloud, and that you have a GCP
+project with Cloud Build enabled. If needed, consult:
+* the [GCP quickstarts][gcp-quickstarts] to setup a GCP project
+* the [cloud build quickstarts][cloud-build-quickstarts] to setup Cloud Build
+  in your project
+
+This guide shows you how to test the builder by using it to create a local
+container. These steps will require you to have [docker] and the
+[pack tool][pack-install] on your workstation.
+
+Verify the [docker tool][docker] is functional on your workstation:
+
+```shell
+docker run hello-world
+# Output: Hello from Docker! and then some more informational messages.
+```
+
+If needed, use the [online instructions][docker-install] to download and install
+this tool. This guide assumes that you have configured [sudoless docker], if
+you prefer replace all `docker` commands below with `sudo docker`.
+
+Verify the [pack tool][pack-install] is functional on our workstation. These
+instructions were tested with v0.17.0, they should work with newer versions.
```

Comment:
s/, they/, although they/

Reply:
Fixed here and other files with similar instructions.